recipes/bletchley/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "bletchley" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://gitlab.com/manny_cyber_wizard/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.bz2
+  sha256: 93ab950e5760855fdba8e071c4cf66bc41568d5c0f647df9729803879d9dfe31
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy >=1.16.0
+    - python
+
+test:
+  imports:
+    - bletchley
+    - bletchley.ciphers
+    - bletchley.cryptanalysis
+
+about:
+  home: https://gitlab.com/manny_cyber_wizard/bletchley
+  license: GNU-3.0
+  license_family: GPL
+  license_file: LICENSE
+  summary: A collection of historical ciphers and cryptanalysis tools
+  doc_url: https://gitlab.com/manny_cyber_wizard/bletchley/wikis/home
+  dev_url: https://gitlab.com/manny_cyber_wizard/bletchley
+
+extra:
+  recipe-maintainers:
+    - marcelotrevisani
+    - mannycyber
```

Comment:
@mannycyber could you please confirm that you are ok to be a maintainer of this recipe as well?

Reply:
That's fine